app/src/main/java/nl/eduvpn/app/service/HistoryService.java

```diff
@@ -44,7 +44,7 @@
  * This allows us to skip some steps, which will make the user experience more fluid.
  * Created by Daniel Zolnai on 2016-10-20.
  */
-public class HistoryService extends Observable {
+public class HistoryService extends LiveData<Void> {
```

Comment:
This is not how the LiveData is intended to be used.
Please convert to a Listener based architecture (interface with the update methods, and then a general listeners array, which can be modified with addListener / removeListener)